stdlib/3/_curses.pyi

```diff
@@ -292,227 +292,171 @@ def use_env(flag: bool) -> None: ...
-
+    def addstr(self, str: str, attr: int = ...) -> None: ...
     @overload
-    def addstr(self, y: int, x: int, str: str, attr: Optional[int]) -> None: ...
-
+    def addstr(self, y: int, x: int, str: str, attr: int = ...) -> None: ...
     def attroff(self, attr: int) -> None: ...
-
     def attron(self, attr: int) -> None: ...
-
     def attrset(self, attr: int) -> None: ...
-
-    def bkgd(self, ch: _chtype, attr: Optional[int]) -> None: ...
-
-    def bkgset(self, ch: _chtype, attr: Optional[int]) -> None: ...
-
-    def border(self, ls: Optional[int], rs: Optional[int],
-               ts: Optional[int], bs: Optional[int],
-               tl: Optional[int], tr: Optional[int],
-               bl: Optional[int], br: Optional[int]) -> None: ...
-
-    def box(self, vertch: Optional[Tuple[int, int]], horch: Optional[Tuple[int, int]]) -> None: ...
-
+    def bkgd(self, ch: _chtype, attr: int = ...) -> None: ...
+    def bkgset(self, ch: _chtype, attr: int = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ..., tl: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ..., tl: _chtype = ..., tr: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ..., tl: _chtype = ..., tr: _chtype = ..., bl: _chtype = ...) -> None: ...
+    @overload
+    def border(self, ls: _chtype = ..., rs: _chtype = ..., ts: _chtype = ..., bs: _chtype = ..., tl: _chtype = ..., tr: _chtype = ..., bl: _chtype = ..., br: _chtype = ...) -> None: ...
+    @overload
+    def box(self) -> None: ...
+    @overload
+    def box(self, vertch: _chtype = ..., horch: _chtype = ...) -> None: ...
     @overload
     def chgat(self, attr: int) -> None: ...
-
     @overload
     def chgat(self, num: int, attr: int) -> None: ...
-
     @overload
     def chgat(self, y: int, x: int, attr: int) -> None: ...
-
     @overload
     def chgat(self, y: int, x: int, num: int, attr: int) -> None: ...
-
     def clear(self) -> None: ...
-
-    def clearok(self, yes: int): ...
-
+    def clearok(self, yes: int) -> None: ...
     def clrtobot(self) -> None: ...
-
     def clrtoeol(self) -> None: ...
-
     def cursyncup(self) -> None: ...
-
     @overload
     def delch(self) -> None: ...
-
     @overload
-    def delch(self, y: int, x: int) -> None: ...
-
+    def delch(self, y: int = ..., x: int = ...) -> None: ...
```

Comment:
I don't think `= ...` is correct here; if one of these is given both must be given (if I read the docs right).

Reply:
Yeah, Thanks for the catch.